nucleus/payara-modules/requesttracing-core/src/test/java/fish/payara/nucleus/requesttracing/RequestTracingServiceMockTest.java

```diff
@@ -103,6 +103,11 @@ public void endingTraceInvokesNotificationServiceSuccessfullyWhenThresholdExceed
 }
 
 class RequestTracingServiceMock extends RequestTracingService {
+    
+    @Override
+    public boolean isTraceInProgress() {
+        return true;
```

Comment:
To improve performance, one of the changes prevents stopTrace from doing anything while a trace isn't in progress. The mock test tests the side effect of the stopTrace method, and the startTrace method can't be run because it's only using mocks. The workaround is overriding the isTraceInProgress in the mock